proxstar/__init__.py

```diff
@@ -247,47 +266,43 @@ def vm_power(vmid, action):
             vm.start()
         elif action == 'stop':
             vm.stop()
-            send_stop_ssh_tunnel(vmid)
+            # TODO (willnilges): Replace with remove target function or something
+            # send_stop_ssh_tunnel(vmid)
```

Comment:
I know you said you would remove these in a future PR, please make an issue so it is documented that this needs to happen.

Reply:
I feel like it's impractical to use an issue to document something that hasn't been committed yet. The TODO _is_ the documentation, and I'm gonna PR for it as soon as this is merged, because this PR is already massive

Reply:
I honestly don't think TODO in code is bad, I tend to see issues as for higher level things like "it crashes when X" rather than "Refactor this particular method"